deps/getopt/getopt.gyp

```diff
@@ -0,0 +1,13 @@
+{
+  'targets': [
+    {
+      'target_name': 'getopt',
+      'type': 'static_library',
+      'sources': [
+        'getopt.h',
+        'getopt.c',
+        'getopt_long.c'
```

Comment:
You can probably drop getopt.c if you build getopt_long.c with -DREPLACE_GETOPT.